ext/bcmath/bcmath.c

```diff
@@ -716,7 +721,8 @@ PHP_FUNCTION(bcround)
 		case PHP_ROUND_AWAY_FROM_ZERO:
 			break;
 		default:
-			zend_argument_value_error(3, "must be a valid rounding mode (PHP_ROUND_*)");
+			/* This is currently unreachable, but might become reachable when new modes are added. */
+			zend_argument_value_error(3, " is a rounding mode that is not supported");
```

Comment:
Leading space in the error message, also I think the more standard wording would either be:

```suggestion
			zend_argument_value_error(3, "is an unsupported rounding mode");
```

or


```suggestion
			zend_argument_value_error(3, "must be a supported rounding mode");
```